ui/plugins/pinboard.js

```diff
@@ -245,5 +249,36 @@ treeherder.controller('PinboardCtrl', [
         $scope.pinnedJobs = thPinboard.pinnedJobs;
         $scope.relatedBugs = thPinboard.relatedBugs;
 
+        var getRevisionTips = function(projectName, list) {
```

Comment:
We already have this information in memory, so you don't need to make a model call to get it.  Just access it like this:

    ThResultSetStore.getResultSetsArray($scope.repoName)